src/core/store.ts

```diff
@@ -48,10 +93,31 @@ type StateListener = () => void
 type MountedAtoms = Set<AnyAtom>
 
 // store methods
+/**
+ * Read an atom's hidden [AtomState]. Derived atom state may be recomputed if they
+ * are invalidated and any of their transitive dependencies have changed.
+ */
 export const READ_ATOM = 'r'
+/**
+ * Write an atom's Value.
```

Comment:
To be precise, it invokes `atom.write` and it doesn't always write *this* atom's value.